runtime/src/mips.rs

```diff
@@ -299,9 +313,37 @@ decl_module! {
             Self::close_proposal(proposal_hash.clone());
         }
 
+        /// An emergency proposal that bypasses network voting process. Governance committee can make
+        /// a proposal that automatically becomes a referendum on which the committee can vote on.
+        #[weight = SimpleDispatchInfo::FixedOperational(200_000)]
+        pub fn emergency_referendum(origin, proposal: Box<T::Proposal>) {
+            // Proposal must originate from the committee
+            T::CommitteeOrigin::try_origin(origin)
+                .map(|_| ())
+                .or_else(ensure_root)
+                .map_err(|_| "bad origin")?;
+
+            let proposal_hash = T::Hashing::hash_of(&proposal);
+
+            // Proposal must be new
+            ensure!(!<Proposals<T>>::exists(proposal_hash), "proposal from committee already exists");
```

Comment:
Does this introduce an attack vector where a "normal" MESH holder could add a proposal ballot ahead of the same proposal coming from an emergency vote, thus stopping the emergency vote going through and forcing the ballot process / timeline?

Reply:
Committee can kill a proposal. In any case, I've added a new method `fast_track_proposal` proposal which promotes a proposal to a referendum. So if the network proposes something that needs to be voted on urgently, committee can fast track that proposal to an immediate vote.